### ISSUE TITLE
Remove console logging

### ISSUE THREAD

**Issue by samtingleff** [digi-trust/dt-cdn#63]
Console logging should never happen in production. People tend to think it represents a problem, etc. We have reports of the following console log:

```
DigiTrust crypto: Invalid browser crypt object
```

**Comment by samtingleff** [comment]
I believe DigiTrustCrypto.initLog() just needs to properly set opts.logging:

```
       if(opts.logging == null){
               opts.logging = configGeneral.logging
       }
```

and call into `initLog()` from early within `isMsCrypto()`

**Comment by samtingleff** [comment]
Released in 1.5.15
